sklearn/cluster/tests/test_optics.py

```diff
@@ -0,0 +1,342 @@
+# Authors: Shane Grigsby <refuge@rocktalus.com>
+#          Amy X. Zhang <axz@mit.edu>
+# License: BSD 3 clause
+
+import numpy as np
+
+from sklearn.datasets.samples_generator import make_blobs
+from sklearn.cluster.optics_ import OPTICS
+from sklearn.utils.testing import assert_equal, assert_warns
+from sklearn.utils.testing import assert_array_almost_equal
+from sklearn.utils.testing import assert_raises
+
+from sklearn.cluster.tests.common import generate_clustered_data
+
+
+def test_optics():
+    # Tests the optics clustering method and all functions inside it
+    # 'auto' mode
+
+    n_clusters = 3
+    X = generate_clustered_data(n_clusters=n_clusters)
+    print(np.shape(X))
+    # Parameters chosen specifically for this task.
+    # Compute OPTICS
+    clust = OPTICS(max_bound=5.0 * 6.0, min_samples=4, metric='euclidean')
+    clust.fit(X)
+    # number of clusters, ignoring noise if present
+    n_clusters_1 = len(set(clust.labels_)) - int(-1 in clust.labels_)
+    assert_equal(n_clusters_1, n_clusters)
+
+
+def test_optics2():
+    # Tests the optics clustering method and all functions inside it
```

Comment:
The comment does not match the test.
`# test that we check a minimum number of samples`

Reply:
fixed.